### DISCUSSION TITLE
New notebook on model calibration available

### DISCUSSION THREAD

**Discussion by raoulcollenteur** [pastas/pastas#290] [Show and tell]
The calibration of Pastas models can be challenging, so I put together a little notebook with some tips and tricks that can help to get a good model fit! Feedback and suggestions welcome! Check out the notebook in the Examples section on the documentation website:

https://pastas.readthedocs.io/

**Comment by Damoche01** [comment]
The link you dropped here does not work. Can you check again and fix it?

**Reply by dbrakenhoff** [reply]
I updated the link.

**Reply by raoulcollenteur** [reply]
I just referred to the docs now :) Should be easy to find in the examples section.

**Reply by Damoche01** [reply]
Thank you. The link works now.
